trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/util/GeneralDataTypeTransformer.java

```diff
@@ -69,6 +69,8 @@ public Type getTrinoType(com.datastrato.gravitino.rel.types.Type type) {
         return VarcharType.createVarcharType(((Types.VarCharType) type).length());
       case BINARY:
         return VARBINARY;
+      case FIXED:
+        return VARBINARY;
```

Comment:
The FIXED type is a GeneralDataType?. Is it appropriate to implement it here?

Reply:
I moved it to `IcebergDataTypeTransformer` as `Fixed` type is only for Iceberg.